.github/workflows/build-test.yml

```diff
@@ -75,6 +75,15 @@ jobs:
           sudo apt-get update
           sudo apt-get install llvm-${{ matrix.llvm }}-dev libclang-${{ matrix.llvm }}-dev
 
+      - name: Install LLVM tools
```

Comment:
does the rustup component `llvm-tools-preview` not include filecheck?

Reply:
seems like no

```
$ docker run --rm -it rust:1.66 bash
root@629bf09695e7:/# rustup component add llvm-tools-preview
info: downloading component 'llvm-tools-preview'
info: installing component 'llvm-tools-preview'
 32.9 MiB /  32.9 MiB (100 %)  22.3 MiB/s in  1s ETA:  0s
root@629bf09695e7:/# FileCheck
bash: FileCheck: command not found
root@629bf09695e7:/# find / -name "*FileCheck*"
root@629bf09695e7:/#
```